algorithms/a3c/test.py

```diff
@@ -0,0 +1,78 @@
+"""
+Adapted from https://github.com/ikostrikov/pytorch-a3c
+"""
+
+import time
+from collections import deque
+
+import torch
+import torch.nn.functional as F
+from torch.autograd import Variable
+
+# from envs import create_atari_env
+from gym_ai2thor.envs.ai2thor_env import AI2ThorEnv
+from model import ActorCritic
```

Comment:
Change the imports relative to root path. 
```suggestion
from algorithms.a3c.model import ActorCritic
```
This applies also in other scripts

Reply:
is 
`from test import test`
wrong too?
Where do you draw the line? Not everything is the full path? relative paths are ok when?

Reply:
`from algorithms.a3c.test import test`
I did this but pycharm gives warning:
`package containing module 'algorithms' is not listed in project requirements`